css/styles.css

```diff
@@ -0,0 +1,884 @@
+    line-height: 14px;
+    padding: 0 12px;
+}
+
+.navBottom ul li a {
+    color: #aaaaaa;
+    text-decoration: none;
+}
+
+.navBottom ul li a:hover {
+    color: #000000;
+}
+
+.fa-search {
+    display: flex;
+    justify-content: flex-end;
+}
+
+.fa-search {
+    color: #aaaaaa;
+    cursor: pointer;
+    border-left: 1px solid #aaaaaa;
+    padding: 15px 20px;
+}
+
+.fa-align-justify:before {
+    display: none;
+}
+
+
+#tnw1, #tnw2 {
+    display: none;
+}
+
+/* Top Content Styling */
+
+.top-content {
+    display: grid;
+    grid-column-gap: 10px;
+    grid-row-gap: 10px;
+    grid-template-columns: 2fr 1fr;
+    grid-template-rows: 243px 243px;
+    margin: 90px 10px 10px 10px;
+}
+
+.top-content p, h1, h2, i {
+    color: white;
+    font-family: sans-serif;
+}
+
+.top-content h1 {
+    font-size: 143.6px;
+    letter-spacing: 14.36px;
+    line-height: 143.6px;
+}
+
+#firstInsight {
+    background: linear-gradient(to bottom right, red,rgba(255, 0, 0, 0.25), black, black ), url("/images/img-1.png");
+    grid-row: 1 / span 2;
+    padding-left: 15px;
+}
+
+.FIP {
+    clear: both;
+    font-size: 14.4px;
+    line-height: 14.4px;
+    text-align: start;
+}
+
+.FIH2 {
+    clear: both;
+    font-size: 46px;
+    line-height: 46px;
+    position: relative;
+    top: 38%;
+}
+
+.FIDate {
+    float: right;
+    font-size: 14.4px;
+    line-height: 14.4px;
+    margin: 15px;
+}
+
+.FITitle {
+    float: left;
+}
+
+.FIAuthor {
+    display: flex;
+    flex-direction: row;
+    position: relative;
+    top: 39.5%;
+}
+
+.FIAPadding {
+    color: lightgray !important;
+    padding: 0 3px;
+}
+
+#secondInsight {
+    align-content: flex-end;
+    background: linear-gradient(to bottom right, red,rgba(255, 0, 0, 0.25), black, black ), url("/images/img-2.jpg");
+    display: grid;
+    padding: 20px;
+}
+
+#thirdInsight {
+    align-content: flex-end;
+    background: linear-gradient(to bottom right, red,rgba(255, 0, 0, 0.25), black, black ), url("/images/img-3.jpg");
+    display: grid;
+    padding: 20px;
+}
+
+.titleInsight {
+    font-size: 28px;
+    line-height: 28px;
+    margin-bottom: 10px;
+}
+
+.authorLeft {
+    display: flex;
+    flex-direction: row;
+}
+
+/* Main */
+
+main {
+    margin: 0 auto;
+    max-width: 1200px;
+}
+
+.latestNews {
+    display: grid;
+    grid-template-columns: repeat(auto-fill, minmax(229px,1fr));
+    grid-gap: 20px;
+    margin-top: 66px;
+    padding: 20px
+}
+
+
+.latestNews h2 {
+    color :#ff3300;
+    font-size: 25.6px;
+    grid-column: 1/ -1;
+    line-height: 25.6px;
+    text-align: start;
+}
+
+.LatestNewsTitle {
+    font-family: "ARS Maquette", "Helvetica Neue", Arial, sans-serif;
+    font-size: 16px;
+    line-height: 20px;
+}
+
+.latestNewsAuthor {
+    color: #aaaaaa;
+    font-size: 13px;
+    line-height: 20px;
+    padding-top: 3px;
+}
+
+.latestNewsFig img {
+    width: 100%;
+}
+
+.subwellArticle {
+    background-color: #fffaf8;
+    /* max-width: 270px; */
```

Comment:
#### `REQUIRED`

Consider deleting commented out codes. If you need to review a previous version of your code, use git history.